Formula/czmqpp.rb

```diff
@@ -0,0 +1,81 @@
+class Czmqpp < Formula
+  desc "C++ wrapper for czmq"
+  homepage "https://github.com/zeromq/czmqpp"
+  url "https://github.com/zeromq/czmqpp.git", :tag => "v1.2.0", :revision => "a3c38eb6e3018887a8ecabc77f3029f7edb7b2d0"
+
+  head "https://github.com/zeromq/czmqpp.git"
+
+  option :universal
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "czmq"
```

Comment:
> You mean I should add something to the `install` block too – how would that work?

It could start off something like this (but as I explained, it is not actually necessary because the `configure` script will do the right thing without it):

```ruby
def install
  ENV.cxx11
  ENV.universal_binary if build.universal?

  # Rest of the code …
```

> Also, I don't see `ENV.cxx11` in the above code.

Sorry, my bad. I just assumed it would be clear that it's sufficient to stick this line somewhere in `test do` *before* invoking the compiler, e.g., like this (that's incomplete test code):

```ruby
ENV.cxx11
args = ENV.cxx.split + ENV.cxxflags.to_s.split + %W[
  -o test test.cpp
  -I#{include} -L#{lib} -lczmq++
  -L#{Formula["czmq"].opt_lib} -lczmq
]
system(*args)
```